githook.py

```diff
@@ -175,7 +175,8 @@ def pre_push_hook(_unused_arg_remote_name, _unused_arg_remote_location):
 
         # Parse files to lint: split at NUL characters, remove blank entries,
         # and remove duplicates.
-        files_to_lint = list({f for f in files_to_lint.split('\0') if f})
+        files_to_lint = list({f for f in files_to_lint.split('\0')
+                              if f and os.path.exists(f)})
```

Comment:
Oh, that's a neat way of handling it. Nice. I was envisioning needing to change the command that gets the list of files in the first place. Nice. 🎉